### DISCUSSION TITLE
Parameter controlling intestinal permeability

### DISCUSSION THREAD

**Discussion by Ricardo-DO** [Open-Systems-Pharmacology/Forum#1740] [Questions & Problems] [answered]
Dear all,

I was looking at one of the semaglutide papers, and I was wondering if the stomach absorption (caused by the permeability enhancer in their formulation) could be simulated in PKSim/MoBi.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/9b068ab1-4695-4efd-b896-478d3d43dc39)
https://pubmed.ncbi.nlm.nih.gov/30429357/

I was checking if there was a way that stomach permeability could be modified but I think that it is not part of the model, as until recently, absorption in stomach was not considered.

Then I was trying to find how to shut down absorption in different regions of the GI tract to see if I could think in a workaround.
I think I was disabling absorption to the mucosa and from the mucosa to blood, but although there is nothing in circulation, the compound is absorbed intracellularly in the intestine (I just checked duodenum, upper and lower jejunum).

I was using one molecule from the templates. I set all the Permeability parameters to zero or to 1E-08  cm/min (just to avoid any problem if there is a division by zero). 

Transcellular was left with its original value
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/a344e6a8-782d-4340-8d55-0c01561751fb)

Mucosas were modified to this
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/bce4ba32-584e-417a-b86b-6f29410e04d4)

Permeabilitites to this
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/530ea7b0-5a4d-42f4-b199-c8d714df1802)

I was expecting no absorption at all in any segment of the intestine, however, there is intracellular concentration in duodenum and fraction absorbed into mucosa up to ~20%.
This was either with zero or with 1E-8 cm/min.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/59565610/e2168f0a-b618-46c4-941f-ddfc8942f132)

Please find attached the PKSim file if of any help, it was generated in PKSim v11.

So my questions are:
1) Is there a way to manipulate gastric absorption in PKSim, or does it require to change many advance parameters in MoBi?
2) Is there a way to stop absorption to the cells of the intestine? Or is it only controlled by the main transcellular Permeabily value? Or the way I tried to shut down permeability is not the correct?

Many thanks,
Ricardo

[Permeability.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/14780598/Permeability.zip)

**Comment by Yuri05** [comment, marked as answer]
>I was checking if there was a way that stomach permeability could be modified but I think that it is not part of the model, as until recently, absorption in stomach was not considered.

Correct.

> Then I was trying to find how to shut down absorption in different regions of the GI tract to see if I could think in a workaround.

You can do it in several ways. 
* If you want to disable absorption in ALL GI segments (not sure though, what's then the rationale for an oral administration...) - you could e.g. set the `Specific intestinal permeability` to zero for your compound.

  <img width="650" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/b1b5773b-7847-4e5c-9ca4-fe1716c3a568">

* if you want to disable absorption in a particular GI segment for **all** compounds - you could e.g. set the <br> `Effective surface area enhancement factor` for those segments to zero

  <img width="650" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/c3e92974-01ef-4b9f-baf7-1fcf183d6f1d">

* if you want to disable absorption in a **particular GI segment for a specific compound** - you could e.g. set the Intestinal solubility to zero 

  <img width="450" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/2f702794-588d-434c-aa2f-37107504b7c4">

>I think I was disabling absorption to the mucosa and from the mucosa to blood, but although there is nothing in circulation, the compound is absorbed intracellularly in the intestine (I just checked duodenum, upper and lower jejunum).

In fact, in the first part you were disabling the flow **within the mucosa**, but not the absorption into the mucosa.
<img width="450" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/217c4955-13db-4171-b811-eccc7b111f8d">

**Comment by Ricardo-DO** [comment]
Hi Juri,

Thanks for your reply.
I was thinking in changing the characteristics of duodenum to make it look like a stomach, then disable all the other GI tract to simulate that setup of the picture with the stomach flow closed.
I think that changing the "Effective surface area enhancement fact" would work better .

Thanks,
Ricardo
